specification/compiler/model/utils.ts

```diff
@@ -489,29 +499,31 @@ export function hoistRequestAnnotations (
 
 /** Lifts jsDoc type annotations to fixed properties on Type */
 export function hoistTypeAnnotations (type: model.TypeDefinition, jsDocs: JSDoc[]): void {
-  const validTags = ['class_serializer', 'url', 'behavior', 'variants', 'variant']
+  const validTags = ['class_serializer', 'doc_url', 'behavior', 'variants', 'variant']
   const tags = parseJsDocTags(jsDocs)
   setTags(type, tags, validTags, (tags, tag, value) => {
     if (tag === 'stability') {
     } else if (tag.endsWith('_serializer')) {
     } else if (tag === 'variants') {
     } else if (tag === 'variant') {
-    } else if (tag === 'url') {
-      type.url = value
+    } else if (tag === 'doc_url') {
+      type.docUrl = value
     } else throw new Error(`Unhandled tag: '${tag}' with value: '${value}' on type ${type.name.name}`)
   })
 }
 
 /** Lifts jsDoc type annotations to fixed properties on Property */
 function hoistPropertyAnnotations (property: model.Property, jsDocs: JSDoc[]): void {
-  const validTags = ['stability', 'prop_serializer', 'url', 'aliases', 'identifier']
+  const validTags = ['stability', 'prop_serializer', 'doc_url', 'aliases', 'identifier', 'url']
```

Comment:
I think you can remove `url` from here.

```suggestion
  const validTags = ['stability', 'prop_serializer', 'doc_url', 'aliases', 'identifier']
```